src/trace/context.spec.ts

```diff
@@ -283,6 +325,42 @@ describe("readTraceFromEvent", () => {
         },
       ],
     });
+    expect(result).toEqual({
```

Comment:
Somehow, I forgot to add an expectation to this test 🤦🏻

Reply:
lol